vignettes/getting-started.Rmd

```diff
@@ -50,7 +50,7 @@ The first data set contains the readcounts from each sample type. Required for a
 
 ```{r}
 example_counts <- example_data %>%
-  dplyr::select(c("Day00_RepA", "Day05_RepA", "Day22_RepA", "Day22_RepB", "Day22_RepC")) %>%
+  select(c("Day00_RepA", "Day05_RepA", "Day22_RepA", "Day22_RepB", "Day22_RepC")) %>%
```

Comment:
Lets get rid of the `pkg::fun` syntax as it makes code hard to read.

Reply:
This is a matter of opinion. I find it easier to read code this way because I know which function we are calling (some packages have functions of the same name). 

I also don't like loading a full package if we only are using one function from it. 

However that being said I'm not opposed to suggestion -- I think in this context using `library(dplyr)` might be the right move for this audience. The main thing is to be consistent.

Reply:
So yes, I agree!